docs/guidance/publish/change_history.md

```diff
@@ -144,21 +156,25 @@ The record now has an additional release, and the compiled and versioned release
 
 ### Implementation
 
-The supplier has started the construction work, and the procuring entity makes the first payment against the contract using their finance system, which shares a common contract identifier with the contract register. 
+The supplier starts the construction work. After a while, the procuring entity makes the first payment to the supplier. The publisher creates a release to document this update in the process.
 
-An OCDS release is published from the finance system using the 'implementation' tag, with details of the payment provided in the `contracts.implementation.transactions` section.
+Up to this point, the publisher used a single source to generate OCDS documents. They store procurement data in a procurement system. Payment data, however, gets stored in a financial system, so do we need to join the data to create a release?
 
-Since the previous releases from the contract register share a common `ocid` with the release from the finance system, it’s possible to update the OCDS record for the contracting process with the new information from the finance system.
+If the systems share common *process identifiers*, there is no need to join data to create a release. The only requisite is to built the `ocid` using the shared identifier.
```

Comment:
I think the mention of joining data might cause confusion, so I tried to find alternative language:

```markdown
So far, the council used a single procurement system to manage the process. The procurement system published the previous OCDS releases.

The council now uses a separate financial system to manage payments. The financial system publishes the new OCDS release.

The procurement system and the financial system share a common contracting process identifier. This means that the two systems can publish releases using the same `ocid`.
```